wordfreq/__init__.py

```diff
@@ -209,18 +209,30 @@ def iter_wordlist(lang, wordlist='combined'):
     return itertools.chain(*get_frequency_list(lang, wordlist))
 
 
-def half_harmonic_mean(a, b):
```

Comment:
...with apologies to Rob for removing this function.  But it seems easier just to use the explicitly associative version.